transport_nantes/transport_nantes/settings.py

```diff
@@ -13,6 +13,8 @@
 from django.urls import reverse_lazy
 from . import settings_local
 import sys
+import os
```

Comment:
Is this needed?  See below.

Reply:
Yes as base_dir isn't in prod (see below)